SPECS/moby-runc/moby-runc.spec

```diff
@@ -1,22 +1,23 @@
 Summary:        CLI tool for spawning and running containers per OCI spec.
 Name:           moby-runc
-Version:        1.0.0~rc10+azure
-Release:        6%{?dist}
+Version:        1.0.0~rc94+azure
+Release:        1%{?dist}
 License:        ASL 2.0
 URL:            https://runc.io/
 
-#Source0:       https://github.com/opencontainers/runc/releases/download/v1.0.0-rc10/runc.tar.xz
-Source0:        runc-v1.0.0-rc10.tar.xz
-#Source1:       https://github.com/sirupsen/logrus/archive/v1.4.1.tar.gz
-Source1:        logrus-v1.4.1.tar.gz
+# See generate-sources.sh for creating runc source tarball
```

Comment:
What do you think about making `RUNC_COMMIT` and `VERSION` input arguments for `generate-sources.sh` and providing the argument values this tarball was generated with in this comment? This way we'd only have to update the spec for a version update and won't be risking keeping an outdated script.

Reply:
All of the moby-* packages have the same type of script, so if we do this I think it should be for all of them at the same time. Will file a bug to track this, but not as part of this PR.